src/org/jetbrains/plugins/scala/project/Versions.scala

```diff
@@ -47,7 +53,10 @@ object Versions  {
   }
 
   private def loadLinesFrom(url: String): Try[Seq[String]] = {
-    Try(HttpConfigurable.getInstance().openHttpConnection(url)).map { connection =>
+    val urlConnection = HttpConfigurable.getInstance().openHttpConnection(url)
```

Comment:
This refactoring is not equivalent to the code you had before if `HttpConfigurable.getInstance().openHttpConnection` can throw. If so, you need to `urlConnection`  to be a `lazy val`.

Reason:
```
scala> import scala.util.Try
import scala.util.Try

scala> Try(throw new Exception)
res0: scala.util.Try[Nothing] = Failure(java.lang.Exception)
```
Throwing an exception inside a `Try.apply` doesn't produce an explosion ;-)